assets/js/vendor.js

```diff
@@ -13772,6 +13772,13 @@ return VueRouter;
 	});
 
 	$.fn.extend({
+    	/*
+		* Replaces space encoded character for serialize function
+		*/
+		weSerialize: function() {
+			return $(this).serialize().replace("%20","+");

```

Comment:
https://make.wordpress.org/core/handbook/best-practices/coding-standards/javascript/#spacing -  “When in doubt, space it out.”
`return $(this).serialize().replace("%20","+");`
`return $( this ).serialize().replace( "%20", "+" );`

----

https://make.wordpress.org/core/handbook/best-practices/coding-standards/javascript/#strings

`return $( this ).serialize().replace( "%20", "+" );`
`return $( this ).serialize().replace( '%20', '+' );`

Reply:
Formatting updated to match coding standards.